Jenkinsfile

```diff
@@ -11,6 +11,11 @@ try {
       checkout scm
       // Make sure not to delete the folder that Jenkins allocates to store scripts
       sh 'git clean -ffdx -e .????????'
+      // Use https url to work around the permission problem
+      sh 'git config --file=.gitmodules submodule.realm/realm-library/src/main/cpp/object-store.url https://github.com/realm/realm-object-store.git'
```

Comment:
Will this just pull in master? Wouldn't it be safer if we specified a specific SHA or tag? Also for reproducibility.